sormas-api/src/main/resources/strings.properties

```diff
@@ -1125,42 +1141,77 @@ messageEventParticipantDearchived = Event participant has been de-archived
 messageEventParticipantResponsibleJurisdictionUpdated = Changing or removing the responsible jurisdiction of this event participant could make you lose access to its personal details and/or disallow you to edit it in the future. Are you sure you want to proceed?
 messageEventParticipantToCaseWithoutEventDisease=It is not possible to create cases from an event participant if the disease of the event has not been set
 messageEventParticipantToContactWithoutEventDisease=It is not possible to create a contact from an event participant if the disease of the event has not been set
 messageEventJurisdictionUpdated = Changing or removing the jurisdiction of this event could make you lose access to its personal details and/or disallow you to edit it in the future. Are you sure you want to proceed?
 messageEventSaved = Event data saved
 messageEventGroupSaved = Event group data saved
-messageEventsNotDeleted = The events mentioned in the above list of UUIDs could not be deleted.
 messageEventsDeleted = All selected eligible events have been deleted
 messageEventsSentToSurvnet = All selected events have been sent to SurvNet
-messageCountCasesNotDeleted = %s cases not deleted. UUIDs of cases not deleted: %s
+
+#change everywhere this message
+messageCountCasesNotArchivedExternalReason = %s cases not archived because the communication with the reporting tool failed: %s
+messageCountCasesNotDearchivedExternalReason = %s cases not dearchived because the communication with the reporting tool failed: %s
+messageCountCasesNotDeleted = %s cases not deleted: %s
+messageCountCasesNotDeletedAccessDeniedReason = %s cases not deleted because they are not in jurisdiction or owned: %s
+messageCountCasesNotDeletedExternalReason = %s cases not deleted because the communication with the reporting tool failed: %s
+messageCountCasesNotDeletedSormasToSormasReason = %s cases not deleted because they were already shared with Sormas to Sormas: %s
 messageCountCasesNotRestored = %s cases not restored. UUIDs of cases not restored: %s
-messageCountContactsNotDeleted = %s contacts not deleted. UUIDs of contacts not deleted: %s
+messageCountContactsNotDeleted = %s contacts not deleted: %s
+messageCountContactsNotDeletedAccessDeniedReason = %s contacts not deleted because they are not in jurisdiction or owned: %s
+messageCountContactsNotDeletedSormasToSormasReason = %s contacts not deleted because they were already shared with Sormas to Sormas: %s
+messageCountContactsNotLinkableAccessDeniedReason = %s contacts not linked because the contact is not editable anymore or the user is missing the rights: %s
 messageCountContactsNotRestored = %s contacts not restored. UUIDs of contact not restored: %s
+messageCountFollowUpsNotCancelled = %s contacts' follow-up not cancelled: %s
+messageCountFollowUpsNotCancelledAccessDeniedReason = %s contacts' follow-up not cancelled because the contact is not editable anymore or is outside the user's jurisdiction: %s
```

Comment:
```suggestion
messageCountVisitsNotCancelledAccessDeniedReason = %s follow-up visits not cancelled because the contact is not editable anymore or is outside the user's jurisdiction: %s
```